application/models/organization.go

```diff
@@ -19,17 +22,51 @@ import (
 	"github.com/silinternational/wecarry-api/domain"
 )
 
-const AuthTypeAzureAD = "azureadv2"
-const AuthTypeGoogle = "google"
-const AuthTypeSaml = "saml"
+type AuthType string
+
+const (
+	AuthTypeAzureAD AuthType = "AZUREADV2"
+	AuthTypeDefault AuthType = "DEFAULT"
```

Comment:
Maybe add a comment about this one.  Does this mean that the organization's authType will be used rather than a special one for an OrgDomain?

Reply:
Yes, it replaces what is now just an empty string. I'll add a comment to clarify. By the way, I didn't extend this to the AuthTypes for user.social_auth_provider. Not sure if that should be defined in the same place or separately.